exporter/awsxrayexporter/README.md

```diff
@@ -1,5 +1,11 @@
 # AWS X-Ray Tracing Exporter for OpenTelemetry Collector
 
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [beta]    |
+| Supported pipeline types | traces    |
+| Distributions            | [contrib] |
```

Comment:
Please include the AWS distro: https://aws-otel.github.io/docs/getting-started/x-ray#configuring-the-aws-x-ray-exporter

Reply:
@willarmiros done